Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 # StackStorm version which refers to Docker images tag
 appVersion: "3.5dev"
```

For the release should this be:
```suggestion
appVersion: "3.5"
```

---

Sounds good to me!